go.mod

```diff
@@ -1,46 +1,49 @@
 module github.com/aws/amazon-cloudwatch-agent
 
-go 1.22.0
-
-toolchain go1.22.2
+go 1.22.4
 
 replace github.com/influxdata/telegraf => github.com/aws/telegraf v0.10.2-0.20240423220441-63baeaedb379
 
+replace (
+	go.opentelemetry.io/collector/config/configgrpc => github.com/amazon-contributing/opentelemetry-collector-contrib/config/configgrpc v0.0.0-20240709194807-b0f0c0eda01b
+	go.opentelemetry.io/collector/config/confighttp => github.com/amazon-contributing/opentelemetry-collector-contrib/config/confighttp v0.0.0-20240709194807-b0f0c0eda01b
+)
+
 // Replace with https://github.com/amazon-contributing/opentelemetry-collector-contrib, there are no requirements for all receivers/processors/exporters
 // to be all replaced since there are some changes that will always be from upstream
 replace (
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter => github.com/amazon-contributing/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter v0.0.0-20240709150923-869060a01b0a
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsemfexporter => github.com/amazon-contributing/opentelemetry-collector-contrib/exporter/awsemfexporter v0.0.0-20240709150923-869060a01b0a
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter => github.com/amazon-contributing/opentelemetry-collector-contrib/exporter/awsxrayexporter v0.0.0-20240709150923-869060a01b0a
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter => github.com/amazon-contributing/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter v0.0.0-20240709194807-b0f0c0eda01b
```

Comment:
This is the hash for your current branch  on contrib repo right? Do we want to keep that hash or the hash that will be on main of contrib repo.

Reply:
this is the new hash off of the latest aws-cwa-dev branch. the update otel dependencies github action didn't work last time I tried